lib/python/Components/Network.py

```diff
@@ -557,7 +557,7 @@ def deactivateInterfaceFinished(self, extra_args):
 		(ifaces, callback) = extra_args
 
 		def checkCommandResult(iface):
-			if self.deactivateInterfaceConsole and self.deactivateInterfaceConsole.appResults.has_key("ifdown " + iface):
+			if self.deactivateInterfaceConsole and "ifdown " + iface in self.deactivateInterfaceConsole.appResults:
```

Comment:
this should be `
if self.deactivateInterfaceConsole and ("ifdown " + iface) in self.deactivateInterfaceConsole.appResults:`

or what you think?

Reply:
You mean having parentheses around the check to make sure it checks both?

I agree with you but is the result any different

Reply:
yes .. i'ts the same result. I have test this.

Reply:
So we're good :+1: 

I need to send this for other distros, this is the first time e2 scene is getting same style.